timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -936,6 +940,88 @@ def evaluate(
             )
         return scores_dict
 
+    def get_feature_importance(
+        self,
+        data: TimeSeriesDataFrame,
+        features: Optional[List[str]] = None,
+        model: Optional[Union[str, AbstractTimeSeriesModel]] = None,
+        metric: Optional[Union[str, TimeSeriesScorer]] = None,
+        time_limit: Optional[float] = None,
+        method: Literal["naive", "permutation"] = "permutation",
+        subsample_size: int = 50,
+        num_iterations: int = 1,
+        random_seed: Optional[int] = None,
+    ) -> pd.DataFrame:
+        assert method in ["naive", "permutation"], f"Invalid feature importance method {method}."
+        metric = check_get_evaluation_metric(metric) if metric else self.eval_metric
```

Comment:
Are we sure that `TimeSeriesScorer` objects are correctly interpreted as booleans? Otherwise we could be safe and check `if metric is not None`